packages/components/menu/src/menu-item.vue

```diff
@@ -19,6 +19,7 @@
       :effect="Effect.DARK"
       placement="right"
       persistent
+      :fallback-placements="['right', 'left', 'bottom', 'top']"
```

Comment:
remove unrelated modifications

Reply:
:fallback-placements="['right', 'left']"

Reply:
I'm not sure. Please point it out

Reply:
> I'm not sure. Please point it out

I don't understand what this change has to do with `select`

Reply:
> remove unrelated modifications

this pr ，fix #6632